plugins/Vestige/Vestige.cpp

```diff
@@ -673,6 +699,8 @@ void VestigeInstrumentView::updateMenu( void )
 void VestigeInstrumentView::modelChanged()
 {
 	m_vi = castModel<VestigeInstrument>();
+	connect(m_vi, SIGNAL(presetChanged()),
+		this, SLOT(changedProgram()));
```

Comment:
You can connect by pointers to member functions in Qt 5.
```suggestion
	connect(m_vi, &VestigeInstrument::presetChanged(),
		this, &VestigeInstrumentView::changedProgram());
```

Reply:
Applied the new syntax